seed/migrations/0084_copy_propertystate_orig_pint.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.6 on 2018-01-11 18:46
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db.models import F
+
+# derived from https://stackoverflow.com/a/41502213/53790
+
+# rgm Jan 2018: The change-over to units-aware columns (ie. everything with
+# `*_pint` as a suffix) is invasive and so inherently a bit risky, as there is a
+# ton of existing data in production with NREL/LBL that makes use of these
+# columns. We'd like to have an escape hatch in case of unforeseen problems,
+# hence we're renaming the existing unitless columns for EUI and areas to be
+# suffixed `*_orig` so that rolling back is a straightforward matter of:
+#
+# - removing the unit-aware application code, and
+# - removing the suffix from the unitless columns
+#
+# Because the `*_pint` columns are new, the forward migration to _start_ using
+# them is straightforward: just copy `*_orig` -> `*_pint` for every
+# PropertyState. This works as a straight float copy because the Pint base
+# units (ft² and kBtu/ft²/year) match the implicit units of the original
+# EUI/area columns.
+
+
+def copy_orig_fields_to_pint_fields(apps, schema_editor):
+    PropertyStateModel = apps.get_model('seed', 'PropertyState')
```

Comment:
I will need to run this locally with a copy of the production db and make sure everything works as expected. It looks right though.